webcompat/api/endpoints.py

```diff
@@ -193,6 +199,15 @@ def proxy_comments(number):
                            mime_type=JSON_MIME_HTML)
 
 
+@api.route('/issues/<int:number>/comments', methods=['GET', 'POST'])
+@mockable_response
+def proxy_comments(number):
+    return proxy_comments_impl(number)
+
+
+proxy_comments_direct = mockable_response_direct(proxy_comments_impl, '/api/issues/{0}/comments')
```

Comment:
This is the line called on `views.py` when we're making a request, per example: `issue_body_json = api.proxy_issue_direct(number)[0]`
What this line does is if we're on normal mode we'll just call the `proxy_comments_impl` and everything will run normally as the old times. But if we're on test mode we'll pass the string `'/api/issues/{0}/comments'` to the new wrapped_function on `helpers.py` and that function will now be able to find the mock outs without using `request.path`.

Reply:
This is weird. This line is in the module by itself in between two functions. 

if it's a specific function you should make it so and call this as a function. Not as a assigned variable in the module. 

This also seems to create a double indirection.

1. `api.proxy_issue_direct(number)[0]` calls …
2. `mockable_response_direct(proxy_issue_impl, '/api/issues/{0}')` calls…
3. `mockable_response_direct` calls…
3. `proxy_issue_impl` calls…
4. `api_request`

It's probably not the best way to do that **but** it shows that if we go this road we need to revise a bit the test strategy which is normal. 

Thanks a lot @marimeireles